versioned_docs/version-8.7/self-managed/modeler/web-modeler/configuration/configuration.md

```diff
@@ -26,16 +26,29 @@ The Camunda 8 [Helm](/self-managed/setup/install.md) and [Docker Compose](/self-
+| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_SCOPE`             | A comma-separated list of the scopes to use with this cluster.                                                 | `test-scope`                                                                |
+| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_AUDIENCE_ZEEBE`    | The permission name for Zeebe.                                                                                 | `zeebe-api`                                                                 |
+| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_AUDIENCE_OPERATE`  | The permission name for Operate.                                                                               | `operate-api`                                                               |
+| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_AUDIENCE_TASKLIST` | The permission name for Tasklist.                                                                              | `tasklist-api`                                                              |
+
+#### Available authentication methods
+
+| Method                                                                                                                                                                                                                  | Description                                                                                                                                                                             | When to use?                                                                                                                                                                                                                         |
+| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `BEARER_TOKEN`<br/><br/>(replaces [deprecated method `OAUTH`](/reference/announcements-release-notes/870/870-announcements.md#deprecated-web-modeler-cluster-authentication-oauth-and-client_credentials-self-managed)) | Web Modeler sends the authenticated user's token with every request to one of the cluster components (Zeebe, Operate, Tasklist).                                                        | The identity provider supports access tokens with multiple audiences.<br/>(_Note_: For the token to be accepted by the different cluster components, it must contain each component's audience.)<br/><br/>Example provider: Keycloak |
```

Comment:
```suggestion
| `BEARER_TOKEN`<br/><br/>(replaces [deprecated method `OAUTH`](/reference/announcements-release-notes/870/870-announcements.md#deprecated-web-modeler-cluster-authentication-oauth-and-client_credentials-self-managed)) | Web Modeler sends the authenticated user's token with every request to one of the cluster components (Zeebe, Operate, Tasklist). | The identity provider supports access tokens with multiple audiences.<br/>For the token to be accepted by the different cluster components, it must contain each component's audience.<br/><br/>Example provider: Keycloak |
```

Reply:
Should we also link to the "generic" OIDC provider config here similar to the next row?

Reply:
Hm, I did not add a link first because Keycloak is the default provider anyway and I did not really find a fitting page to link to. Maybe http://localhost:3000/docs/8.7/self-managed/setup/guides/using-existing-keycloak/ would be more relevant than the "generic" OIDC provider page, but it's also not needed when you just use the Keycloak instance that is installed by default.

Reply:
Yep, makes sense.

Reply:
Regarding the other suggested change: I'd actually prefer to keep the sentence as a note in parentheses because it does not directly answer the "When to use?" question, but provides contextual information.

Reply:
Makes sense, let's keep it 👍